linkerd/docs/telemetry.md

```diff
@@ -54,6 +54,30 @@ Exposes admin endpoints:
 
 This telemeter has no additional parameters.
 
+## Istio (experimental)
+
+> Example Istio config
+
+```yaml
+telemetry:
+- kind: io.l5d.istio
+  experimental: true
+  hostname: 127.0.0.1
+  port: 9091
```

Comment:
might consider naming these `mixerHost` `mixerPort` to be explicit (the other istio components need different host/port combos so I could see this getting confusing)